core-processor/src/processor.rs

```diff
@@ -122,9 +120,13 @@ fn process_error(
     }
 
     if !dispatch.is_reply() || dispatch.exit_code().expect("Checked before") == 0 {
-        let id = MessageId::generate_reply(dispatch.id(), crate::ERR_EXIT_CODE);
-        let packet = ReplyPacket::system(err.encode(), crate::ERR_EXIT_CODE);
-        let message = ReplyMessage::from_packet(id, packet);
+        // # Safty.
```

Comment:
Typo and please no period in the title.
```suggestion
        // # Safety
```